subprojects/build.gradle.kts

```diff
@@ -359,3 +378,21 @@ fun Project.configureJunit5Tests() {
         maxParallelForks = 8
     }
 }
+
+val detektAll by tasks.registering(Detekt::class) {
+    description = "Runs over whole code base without the starting overhead for each module."
+    parallel = true
+    buildUponDefaultConfig = false
+    setSource(files(projectDir))
+    config.setFrom(files(project.rootDir.resolve("detekt.yml")))
+    include("**/*.kt")
+    include("**/*.kts")
+    exclude("**/resources/**")
+    exclude("**/build/**")
+    reports {
+        xml.enabled = false
+        html.enabled = false
```

Comment:
Would it be easier to read html report in a build verdict?

Reply:
don't know how to add it there; just using logs for now